pkg/apis/core/types_seed.go

```diff
@@ -280,6 +280,8 @@ type SeedSettingVerticalPodAutoscaler struct {
 	// is enabled by default because Gardener heavily relies on a VPA being deployed. You should only disable this if
 	// your seed cluster already has another, manually/custom managed VPA deployment.
 	Enabled bool
+	// FeatureGates is a map of feature gates for the vpa components.
```

Comment:
Cosmetic nit: Let's sync the doc string of the field in the internal type (`pkg/apis/core/types_seed.go`) and in the external type (`pkg/apis/core/v1beta1/types_seed.go`):

```suggestion
	// FeatureGates contains information about enabled feature gates.
```

Reply:
Thanks for the suggestion, updated the internal `Seed` API version to match the `FeatureGates` docstring from the external version.